### PR TITLE
chore(settings): set nOSNet as default fallback

### PR DESCRIPTION
**Pull request by DalderupMaurice**
## Description
removed hardcoded fallbacks 
set fallback to nOSNet

## Motivation and Context
Many reasons

## How Has This Been Tested?
Cleared out networks after adding one (one way triggering the fallback)

## Types of changes
- [x] Chore (tests, refactors, and fixes)
- [ ] New feature (adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
- [x] I have read the **[CONTRIBUTING](./CONTRIBUTING.md)** guidelines and confirm that my code follows the code style of this project.
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## Closing issues
Fixes #

### CODE REVIEW

File: src/renderer/settings/components/NetworkPanel/NetworkPanel.js [hunk 1]
```diff
@@ -90,7 +90,7 @@ export default class NetworkPanel extends React.Component {
   handleClearNetworks = () => {
     // If user is on a custom network, switch to TestNet
     if (!PREDEFINED_NETWORKS.includes(this.props.currentNetwork)) {
-      this.props.setCurrentNetwork('TestNet');
+      this.props.setCurrentNetwork(DEFAULT_NET);
```
Comment (mhuggins) [inline src/renderer/settings/components/NetworkPanel/NetworkPanel.js]: Nice find!

File: src/renderer/settings/values/networks.js [hunk 2]
```diff
@@ -1,8 +1,9 @@
 import { settings } from '@cityofzion/neon-js';
 import { keys } from 'lodash';
 
-export const NOS_LOCAL = 'nOSLocal';
-export const NOS_TESTNET = 'nOSNet';
+export const DEFAULT_NET = 'nOSNet';
 
+export const NOS_TESTNET = 'nOSNet';
+export const NOS_LOCAL = 'nOSLocal';
```
Comment (mhuggins) [inline src/renderer/settings/values/networks.js]: Just so we're not redefining the same string twice, can we change this to:

```js
export const NOS_TESTNET = 'nOSNet';
export const NOS_LOCAL = 'nOSLocal';

export const DEFAULT_NET = NOS_TESTNET;
```

File: src/renderer/shared/hocs/withLoadingProp.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ import { withProgress, progressValues } from 'spunky';
 
 const { LOADING } = progressValues;
 
-const PROGRESS_PROP: string = '__progress__';
```
Comment (mhuggins) [inline src/renderer/shared/hocs/withLoadingProp.js]: 😬

File: src/renderer/settings/values/networks.js (hunk 2 above)
Comment (DalderupMaurice) [reply src/renderer/settings/values/networks.js]: I was thinking the same when I wrote it 😂 Didn't persist on doing it hehe

**Pull request merged**